cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -230,6 +230,63 @@ def single_qubit_randomized_benchmarking(
     return RandomizedBenchMarkResult(num_clifford_range, gnd_probs)
 
 
+def parallel_single_qubit_randomized_benchmarking(
+    sampler: 'cirq.Sampler',
+    qubits: Iterator['cirq.Qid'],
+    use_xy_basis: bool = True,
+    *,
+    num_clifford_range: Sequence[int] = cast(
+        Sequence[int], np.logspace(np.log10(5), 3, 5, dtype=int)
```

Comment:
`np.logspace(np.log10(5), 3, 5, dtype=int)` is an `np.ndarray`. I think there are two options here

1. `num_clifford_range: Sequence[int] = np.logspace(np.log10(5), 3, 5, dtype=int).tolist()`
2. `num_clifford_range: Sequence[int] = tuple(np.logspace(np.log10(5), 3, 5, dtype=int))`

personally I would go with the second option to make sure the default option is immutable.